cluster/validate-cluster.sh

```diff
@@ -32,15 +32,24 @@ attempt=0
 while true; do
   # The "kubectl get nodes" output is three columns like this:
   #
-  #     NAME                     LABELS    STATUS
-  #     kubernetes-minion-03nb   <none>    Ready
+  #     NAME                     LABELS    STATUS   AGE
+  #     kubernetes-minion-03nb   <none>    Ready    10m
   #
   # Echo the output, strip the first line, then gather 2 counts:
   #  - Total number of nodes.
   #  - Number of "ready" nodes.
   "${KUBE_ROOT}/cluster/kubectl.sh" get nodes > "${MINIONS_FILE}" || true
   found=$(cat "${MINIONS_FILE}" | sed '1d' | grep -c .) || true
-  ready=$(cat "${MINIONS_FILE}" | sed '1d' | awk '{print $NF}' | grep -c '^Ready') || true
+
```

Comment:
Not your fault, but this is not a good idea. Scripts should:
1. specify --api-version, and
2. use -o template

The human-oriented output format may change at any time.

cc @fabioy